fdbclient/Knobs.h

```diff
@@ -129,6 +129,7 @@ class ClientKnobs : public Knobs {
 	double BACKUP_RANGE_TIMEOUT;
 	double BACKUP_RANGE_MINWAIT;
 	int BACKUP_SNAPSHOT_DISPATCH_INTERVAL_SEC;
+	int BACKUP_INIT_SNAPSHOT_INTERVAL_SEC;
```

Comment:
As @satherton  mentioned, this should not be a knob, instead it should be an argument passed to `submitBackup`.

Reply:
PTAL.

Reply:
If "PTAL" means "please take another look", then please see my comment in `backup.actor.cpp`.  This knob should be removed.